Boop/Boop/scripts/SortAscending.js

```diff
@@ -0,0 +1,16 @@
+/**
+	{
+		"api":1,
+		"name":"Sort Ascending",
+		"description":"Sorts the lines ascending.",
+		"author":"andipaetzold",
+		"icon":"metamorphose",
+		"tags":"sort,ascending,asc"
+	}
+**/
+
+function main(input) {
+
+	input.text = input.text.split('\n').sort().join('\n')
```

Comment:
Would be nice if it supported non-ASCII characters using `String.localeCompare`:

```diff
- input.text = input.text.split('\n').sort().join('\n')
+ input.text = input.text.split('\n').sort(a => a.localeCompare(b)).join('\n')
```

this way it will also correctly sort accented characters and other locale-specific characters in the correct order.

Reply:
Didn't think about those special characters - thanks

pushed the change